### PR TITLE
Add func test to stdci

### PR DESCRIPTION
**Pull request by SchSeba**
Allow the STDCI to run our functional tests.

### CODE REVIEW

File: automation/check-patch.sh [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-    echo "TODO: Add functional tests"
+    echo "Start ovs process"
+    /usr/share/openvswitch/scripts/ovs-ctl --system-id=random start
+
+    cd ..
+    mkdir -p go/src/kubevirt.io
+    mkdir -p go/pkg
+
+    ln -s $(pwd)/ovs-cni go/src/kubevirt.io/
+    ls -la go/src/kubevirt.io/ovs-cni
```
Comment (phoracek) [inline automation/check-patch.sh]: you can drop this

File: automation/check-patch.sh [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+
+    export GOPATH=$(pwd)/go
```
Comment (phoracek) [inline automation/check-patch.sh]: move this right after line 9

File: automation/check-patch.sh [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+    cd go/src/kubevirt.io/ovs-cni
+
+    echo "Run tests"
+    make && make test
```
Comment (phoracek) [inline automation/check-patch.sh]: if we are already on root, you can do `make build test`

File: automation/check-patch.packages [hunk 4]
```diff
@@ -2,3 +2,4 @@ docker
 make
 git
 go
+openvswitch
```
Comment (phoracek) [inline automation/check-patch.packages]: please add a newline after this

File: automation/check-patch.sh (hunk 2 above)
Comment (SchSeba) [reply automation/check-patch.sh]: Done

**Pull request merged**